.github/workflows/stale_bot.yaml

```diff
@@ -0,0 +1,19 @@
+name: Stale bot
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Mark and close stale PRs
+        uses: actions/stale@v4
+        with:
+          stale-pr-message: "This PR is stale because it has been 60 days with no activity. Remove stale label or this will be closed in 7 days."
```

Comment:
This makes it sound like the author can remove the label themselves, which can be confusing.

I suggest changing the 2nd sentence to `This PR will be automatically closed within 7 days if there is no further activity` or similar?

Reply:
In libtorrent, a comment from anyone removes the label. Is it the same here? Probably, because there is no special instructions about it in libtorrent https://github.com/arvidn/libtorrent/blob/RC_2_0/.github/stale.yml which makes the comment to "Remove stale label" inaccurate as it implies that only team members can do that.

IMO only the author and the devs should be able to un-stale a PR but alas, i saw no such option in the bot's documentation. Maybe [remove-pr-stale-when-updated](https://github.com/marketplace/actions/close-stale-issues#remove-pr-stale-when-updated) to false? I would say no for now because the author wouldn't be able to remove the label either. Unless un-staleing by randoms becomes common of course.

Reply:
> Unless un-staleing by randoms becomes common of course.

It's par for the course with issues, but I wouldn't expect this to be common in PRs.

Reply:
>I suggest changing the 2nd sentence to This PR will be automatically closed within 7 days if there is no further activity or similar?

Updated.

>However, before this is merged, we should wait for @sledgehammer999's input

I'll ask him to give a final approval before merging once we are finished here.